gems/ibm_cloud_power/lib/ibm_cloud_power/models/create_image.rb

```diff
@@ -186,8 +186,7 @@ def valid?
       return false if @source.nil?
       source_validator = EnumAttributeValidator.new('String', ["root-project", "url"])
       return false unless source_validator.valid?(@source)
-      os_type_validator = EnumAttributeValidator.new('String', ["aix", "ibmi", "redhat", "sles"])
```

Comment:
@iv1111 please take a look and let me know if I'm breaking anything with the PVC -> PVS image import workflow.

In the image import workflow I think we just use the PVC image `osType` value, which isn't constrained to the PVS `CreateImage` `osType` enum. If I remove this validator then we'll pass any value to the PVS `CreateImage` API and let them tell us if it's good or not. Let me know if you see any problems with this.
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/template.rb#L120
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/image_import_workflow.rb#L17

Reply:
Hi. My understanding is that when calling _create_image_ the validator will check if the passed value is from the above list of acceptable OS names and will reject the call if it's not. This should not break the image-import code. If the os type provided during the image import workflow is not from that list then the workflow would fail on the API remote anyway.  The valid list is ["aix", "ibmi", "**rhel**", "sles"]: https://cloud.ibm.com/apidocs/power-cloud#pcloud-v1-cloudinstances-cosimages-post